applications/main/bad_usb/bad_usb_script.c

```diff
@@ -109,8 +109,8 @@ static const char ducky_cmd_delay[] = {"DELAY "};
 static const char ducky_cmd_string[] = {"STRING "};
 static const char ducky_cmd_defdelay_1[] = {"DEFAULT_DELAY "};
 static const char ducky_cmd_defdelay_2[] = {"DEFAULTDELAY "};
-static const char ducky_cmd_stringdelay_1[] = {"STRINGDELAY"};
-static const char ducky_cmd_stringdelay_2[] = {"STRING_DELAY"};
+static const char ducky_cmd_stringdelay_1[] = {"STRINGDELAY "};
```

Comment:
forgot to mention, for some reason commands have a space after them, reformatted addition too